index.js

```diff
@@ -1,3 +1,4 @@
 export { default as Swipeable } from './Swipeable';
 export { default as DrawerLayout } from './DrawerLayout';
 export * from './GestureHandler';
+export * from './touchables';
```

Comment:
@osdnk this breaks the latest installation as the folder `touchables` is in Examples

Reply:
Sorry for the trouble @zocoi we noticed that few seconds after publishing 1.0.13 and 1.0.14 is already published on npm and includes missing files

Reply:
@kmagiera will you cut a new release soon? Or can I use an older version?

Reply:
As I mentioned – 1.0.14 is already out and it fixes the problem